### PR TITLE
Update default target platform version - net6.0

### PR DESCRIPTION
**Pull request by JoonghyunCho**
- Update the default target platform version to `7.0`.
- `7.0` is the latest public released target platform version.
- Update `KnownFrameworkReference` to target the `TargetFramework`

### CODE REVIEW

**Review by sung-su — APPROVED** [review]

**Comment by sung-su** [comment]
Please fix net7.0 net8.0 branchs too

File: workload/src/Samsung.Tizen.Sdk/targets/Samsung.Tizen.Sdk.Versions.targets [hunk 1]
```diff
@@ -49,7 +49,7 @@ Copyright (c) Samsung All rights reserved.
   <ItemGroup>
     <KnownFrameworkReference
       Include="Samsung.Tizen"
-      TargetFramework="net6.0-tizen$(_DefaultTargetPlatformVersion)"
+      TargetFramework="$(TargetFramework)"
```
Comment (rookiejava) [inline workload/src/Samsung.Tizen.Sdk/targets/Samsung.Tizen.Sdk.Versions.targets:52]: need to consider the use of implcit tpm version. e.g., `net6.0-tizen`.

Comment (JoonghyunCho) [reply workload/src/Samsung.Tizen.Sdk/targets/Samsung.Tizen.Sdk.Versions.targets:53]: Thanks for pointing that out! I added the code supporting that.

**Review by sung-su — APPROVED** [review]
LGTM

**Pull request merged**